README.md

```diff
@@ -171,7 +171,17 @@ TRIE.CONTAINS checks if a string exists in a given trie key. If the trie key doe
        (int) 0
        redis> TRIE.CONTAINS key1 hello
        (int) -1
-
+       
+### TRIE.COMPLETIONS key value
+TRIE.COMPLETIONS returns the number of possible endings a given prefix has in a given trie key. If the key does not previously exist, an error will be thrown. Otherwise, the command will return either an integer with the number of possible endings a prefix has, or, if the prefix does not exist in the trie, will return 0.
```

Comment:
Awkward wording. "Otherwise, the command will return the number of possible endings the prefix has. In the case or the prefix not being in the trie, the command will return 0."

Reply:
Also, I'm curious about the case of a "prefix" which is also a word. For example, for key1 containing only ball, would TRIE.COMPLETIONS key1 ball return 1 or 0? I feel like it'd be useful to have such an example be included below.